src/core/maps/grid_cell_factory.h

```diff
@@ -1,10 +1,21 @@
+/**
+ * \file
+ * There are classes in this file:
+ * GridCell - abstract base class, whose subclasses ;
+ * GridCellFactory - abstract base class, whose subclasses create a new cell.
+ */
+
 #ifndef __GRID_CELL_FACTORY_H
 #define __GRID_CELL_FACTORY_H
 
 #include <memory>
 
 #include "cell_occupancy_estimator.h"
 
+/**
+ * \brief This is abstract class. Derived classes of this class calculate the probability that a cell is occupied.
```

Comment:
Should be smth like this: "The base class for GridMap's cell that holds information about occupancy of the cell."